### PR TITLE
Remove outdated link and and new ones for the gen3 Storj DCS

### PR DESCRIPTION
**Pull request by elek**
This project has not been updated recently, but I think it would be useful (and awesome ;-) ) to maintain it. Can make easier to find the right resources to use Storj DCS

### CODE REVIEW

**Review by ifraixedes — APPROVED** [review]

**Review by kristaxox — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,102 +1,72 @@
...
+
+ ## Clients with Storj support and integrations 
+
+* **[uplink](https://github.com/storj/storj/releases) - Uplink is the official CLI for Storj. Can be downloaded from the release page** 
+* [Storj Mobile](https://play.google.com/store/apps/details?id=com.storj_mobile) (beta) - Android application to access Storj ([forum post](https://forum.storj.io/t/storj-mobile-beta-for-android/15578))
+* [rclone](https://rclone.org/tardigrade/) - Tool to sync your files to cloud storage including Storj 
+* [restic]() - Restic is a modern backup program. Supports Storj via the rclone . [howto](https://docs.storj.io/dcs/how-tos/backup-with-restic), [docs](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html#other-services-via-rclone)
+
+* [FileZilla](https://docs.storj.io/dcs/how-tos/set-up-filezilla-for-decentralized-file-transfer) - New protocol for Storj is being added to FileZilla. 
+* [NextCloud](https://github.com/storj-thirdparty/nextcloud-app) - Adds Storj external storage to NextCloud
```
Comment (stefanbenten) [inline README.md]: Should we straight link to this? https://apps.nextcloud.com/apps/storj

File: README.md [hunk 2]
```diff
@@ -1,102 +1,72 @@
...
+
+* [FileZilla](https://docs.storj.io/dcs/how-tos/set-up-filezilla-for-decentralized-file-transfer) - New protocol for Storj is being added to FileZilla. 
+* [NextCloud](https://github.com/storj-thirdparty/nextcloud-app) - Adds Storj external storage to NextCloud 
+* [Duplicati](https://www.duplicati.com/) - Free backup software to store encrypted backups online ([howto](https://docs.storj.io/dcs/how-tos/backup-with-duplicati))
+
+# Client Libraries & language bindings
+
+Storj can be used via the native interface (uplink CLI and uplink libraries) or as an S3 compatible backend with any S3 compatible tool. The following list focuses on the native protocol.
+
+- [uplink](https://github.com/storj/uplink) - Go library to use Storj
```
Comment (stefanbenten) [inline README.md]: Wondering if we should not redirect to either our go docs page for easier reading?

File: README.md [hunk 3]
```diff
@@ -1,102 +1,72 @@
...
-- [Storj on Github](https://github.com/storj) - All Storj repositories in one place.
-- [Give Feedback](https://wantoo.io/storj-product-feedback/) - Suggest new product ideas or upvoting existing ones.
-- [Write Guides](https://storj.io/get-paid-to-write.html) - Get paid to write guides and tutorials about our platform.
-- [Submit Code](https://storj.io/developers.html) - Submit pull requests on our public repositories on GitHub.
+## Services based on / with support of Storj 
+
+* [Fastly](https://fastly.com) - Content delivery network with Storj support ([docs](https://docs.fastly.com/en/guides/storj-dcs-object-storage))
+* [Filebase](https://filebase.com/) - S3 compatible Object Storage. One of the supported storage backend is Storj ([docs](https://docs.filebase.com/what-is-filebase/our-ecosystem/storage-networks/storj#how-do-i-store-data-on-storj-through-filebase))
+* [Opensea](https://opensea.io/) -- NFT marketplace. Supports hosting NFT resources on Storj ([howto](https://docs.storj.io/dcs/how-tos/nft-storage))
+
```
Comment (stefanbenten) [inline README.md:56]: Arent we missing Pixelexperience here that use us to distribute their roms?

File: README.md [hunk 4]
```diff
@@ -1,102 +1,72 @@
...
-- [StorjDotNet](https://github.com/ssa3512/StorjDotNet) - .NET library for interacting with Storj (work-in-progress)
-- [libstorj.NET](https://github.com/TopperDEL/libstorj.NET) - Working .NET library based on java-libstorj, nuget-package available (work-in-progress)
-- [Storj.js](https://github.com/Storj/storj.js) - Browser library for interacting with Storj (work-in-progress)
-- [Storj Python Library](https://github.com/storj/storj-python-sdk) - Python library for interacting with Storj (work-in-progress)
-- [Storj-PHP](https://github.com/WebWeave/storj-php) - Implementation of the Storj protocol for PHP (work-in-progress)
+- **[Start here](https://www.storj.io/) - Register on satellites operated by Storj Labs**
+- [Documentation](https://docs.storj.io/dcs/) - Entrypoint to the official documentation
+- [Community support / forum](forum.storj.io/) - Community and support forum
+- [Storj Status](https://status.storj.io/) - Status of the Storj network
+- [Usage statistic dashboard](https://storjstats.info/) - Statistics about the full Storj network
```
Comment (stefanbenten) [inline README.md]: Should we also link to the direct API here?

File: README.md (hunk 1 above)
Comment (elek) [reply README.md]: Yes, that's definitely better, thanks.

File: README.md (hunk 3 above)
Comment (elek) [reply README.md:56]: Sure, I added them.

Long-term we may need to differentiate between the services who use Storj under the hood and the services who offer Storj as an option. But as I am writing this I just realized that how thin is this difference ;-)

File: README.md (hunk 2 above)
Comment (elek) [reply README.md]: Good point, but I am not sure if we have good generated documentation for all the bidings. We have it for uplink. I would add the documentation link to the end of the line (and later I can replace it with a small `badge`-link link.) What do you think?

Comment (stefanbenten) [reply README.md]: That sounds good to me!

**Review by stefanbenten — APPROVED** [review]

**Pull request closed without merging**